### PR TITLE
docs: Proposal of expense 0.1.0 format

### PR DESCRIPTION
**Pull request by kevindavee**
## Description of the changes
Expense version `0.1.0` format proposal.

### CODE REVIEW

**Comment by kevindavee** [comment]
Let me know or feel free to add more relevant reviewer @benjlevesque @yomarion

File: packages/data-format/src/format/rnf_expense/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| version | constant | Mandatory | value: "0.1.0"       |
+
+## employeeInfo
+
+_Information about the employee_
+
+| Name       | Type                     | Need      | Comment            |
+| ---------- | ------------------------ | --------- | ------------------ |
+| email      | string _(format: email)_ | Mandatory | email              |
+| employeeId | string                   | Mandatory | id of the employee |
```
Comment (yomarion) [inline packages/data-format/src/format/rnf_expense/README.md:29]:
```suggestion
```
This does not belong to the RN protocol.

File: packages/data-format/src/format/rnf_expense/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+## employerInfo
+
+_Information about the employer_
+
+| Name                | Type                     | Need      | Comment                                                                                                          |
+| ------------------- | ------------------------ | --------- | ---------------------------------------------------------------------------------------------------------------- |
+| email               | string _(format: email)_ | Mandatory | email                                                                                                            |
+| firstName           | string                   | Mandatory | first name                                                                                                       |
+| lastName            | string                   | Mandatory | last name                                                                                                        |
+| businessName        | string                   | Mandatory | business name                                                                                                    |
```
Comment (yomarion) [inline packages/data-format/src/format/rnf_expense/README.md:42]: I think that from a protocol point of view we can make mandatory information minimal. Same applies to `employeeInfo`

File: packages/data-format/src/format/rnf_expense/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+| companyRegistration | string                   | Optional  | company registration                                                                                             |
+| miscellaneous       | object                   | Optional  | Miscellaneous                                                                                                    |
+
+## expenseItem
+
+_List of the items of the expense_
+
+| Name        | Type      | Need      | Comment                                    |
+| ----------- | --------- | --------- | ------------------------------------------ |
+| name        | string    | Mandatory | name of the item                           |
```
Comment (yomarion) [inline packages/data-format/src/format/rnf_expense/README.md:54]: Same comment here, from a protocol point of view, we can have items with no names.

File: packages/data-format/src/format/rnf_expense/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Expense format
+
+| Name            | Type      | Need      | Comment                                |
+| --------------- | --------- | --------- | -------------------------------------- |
+| meta            | object    | Mandatory | see below "meta"                       |
+| creationDate    | date-time | Mandatory | Creation date of the request           |
+| periodStartDate | date-time | Mandatory | Starting period of the current expense |
+| periodEndDate   | date-time | Mandatory | End period of the current expense      |
```
Comment (yomarion) [inline packages/data-format/src/format/rnf_expense/README.md:8]: Here I'm not sure, are both columns really mandatory, in the RN protocol? Maybe not `periodStartDate` ? I'm not sure.

**Comment by kevindavee** [comment]
Will close this first. As discussed with @benjlevesque , we better iterate on the `portal` entity first before deciding if we gonna introduce a new data format in the protocol.

@yomarion , feel free to challenge the data format here
https://github.com/RequestNetwork/portal/blob/81e49c32896658d7e18c43de05a80135728ce97f/src/expenses/entity.ts

**Pull request closed without merging**